src/applications/disability-benefits/all-claims/components/NewDisability.jsx

```diff
@@ -1,22 +1,11 @@
 import React from 'react';
-import { capitalizeEachWord } from '../utils';
 import { NULL_CONDITION_STRING } from '../constants';
-import { showRevisedNewDisabilitiesPage } from '../content/addDisabilities';
 
 export default function NewDisability({ formData }) {
-  if (showRevisedNewDisabilitiesPage()) {
-    return (
-      <div className="vads-u-flex--fill vads-u-padding-right--2 word-break capitalize-first-letter">
-        {typeof formData?.condition === 'string'
-          ? formData.condition
-          : NULL_CONDITION_STRING}
-      </div>
-    );
-  }
   return (
     <div className="vads-u-flex--fill vads-u-padding-right--2 word-break">
```

Comment:
Do we need the `capitalize-first-letter` class here as well?

Reply:
Thanks for catching this; I added the class back in.